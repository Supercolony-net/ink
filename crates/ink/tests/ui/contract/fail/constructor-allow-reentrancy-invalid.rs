#[ink::contract]
mod contract {
    #[ink(storage)]
    pub struct Contract {}

    impl Contract {
        #[ink(constructor, allow_reentrancy)]
        pub fn constructor() -> Self {
            Self {}
        }

        #[ink(message)]
        pub fn message(&self) {}
    }
}

fn main() {}
