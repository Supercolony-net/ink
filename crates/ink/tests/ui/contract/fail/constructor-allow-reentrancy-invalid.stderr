error: encountered conflicting ink! attribute argument
 --> tests/ui/contract/fail/constructor-allow-reentrancy-invalid.rs:7:28
  |
7 |         #[ink(constructor, allow_reentrancy)]
  |                            ^^^^^^^^^^^^^^^^
